cli/command/environment_test.go

```diff
@@ -212,6 +212,60 @@ func TestDeleteEnvironment(t *testing.T) {
 	})
 }
 
+func TestDeleteEnvironmentDependencies(t *testing.T) {
```

Comment:
This isn't testing what the name suggests; currently, you don't assert that we call delete on each dependent entity for that environment.